atox/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="AvatarImageView">
+        <attr name="statusIndicatorVisibility" format="boolean" />
```

Comment:
This should probably be called `statusIndicatorVisible` or reuse the Visibility "enum" that other Android views use, but that would probably be more complex for no reason since it handles collapsing/removing views from the view hierarchy as well.

Reply:
Yeah I will just use `statusIndicatorVisible`.